packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.controller.js

```diff
@@ -1,11 +1,30 @@
 import last from 'lodash/last';
-import { DEDICATED_CLOUD_DATACENTER } from './dedicatedCloud-datacenter.constants';
+import {
+  DEDICATED_CLOUD_DATACENTER,
+  COMMERCIAL_RANGE_NAME_EOL,
+  VDC_MIRGRATION_GUIDE_LINK,
+} from './dedicatedCloud-datacenter.constants';
 
 export default class {
   /* @ngInject */
-  constructor($translate) {
+  constructor($translate, ovhManagerPccDatacenterService) {
     this.$translate = $translate;
     this.DEDICATED_CLOUD_DATACENTER = DEDICATED_CLOUD_DATACENTER;
+    this.ovhManagerPccDatacenterService = ovhManagerPccDatacenterService;
+  }
+
+  $onInit() {
+    this.ovhManagerPccDatacenterService
+      .getCommercialRangeName(
+        this.datacenter.model.serverName,
+        this.datacenter.model.id,
+      )
+      .then((data) => {
+        this.isEOLDatacenter = COMMERCIAL_RANGE_NAME_EOL.includes(data);
+        this.guideMigration =
+          VDC_MIRGRATION_GUIDE_LINK[this.$translate.use()] ||
```

Comment:
Why using local instead of customer subs ?

Reply:
I use `$translate.use()` because it is already imported while `user.ovhSubsidiary` is not.

Reply:
I guess we have to be ISO with current behavior accross Manager.
we use the sub, i think it's better for customers experience.

it's point of view

Reply:
It's applied.